src/components/App/App.component.jsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
 
-import AuthProvider from '../../providers/Auth';
+// import AuthProvider from '../../providers/Auth';
+import SearchProvider from '../../providers/Search';
+import ThemeProvider from '../../providers/Theme';
```

Comment:
Instead of having two providers, you can have a GlobalContext provider so you don't repeat code, and with the useReducer hook, you can handle the reads and writes on the global state in a more organized way.
https://dev.to/jackent2b/the-best-couple-usecontext-usereducer-4e65